model/testing/src/icon4py/model/testing/datatest_utils.py

```diff
@@ -31,8 +31,12 @@
 GAUSS3D_EXPERIMENT = "gauss3d_torus"
 WEISMAN_KLEMP_EXPERIMENT = "weisman_klemp_torus"
 
+MC_CH_R04B09_DSL_GRID_URI = "https://polybox.ethz.ch/index.php/s/hD232znfEPBh4Oh/download"
```

Comment:
can you use the URIs from the definitions.py ? this file is deprecated and we should not add any new things to it.